test/testdata/generate.sh

```diff
@@ -96,13 +96,10 @@ function scaffold_test_project {
       $kb create webhook --version v1 --kind Lakers --defaulting --programmatic-validation
     fi
   elif [[ $project =~ addon ]]; then
-    header_text 'enabling --pattern flag ...'
-    export KUBEBUILDER_ENABLE_PLUGINS=1
     header_text 'Creating APIs ...'
-    $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false --pattern=addon
-    $kb create api --group crew --version v1 --kind FirstMate --controller=true --resource=true --make=false --pattern=addon
-    $kb create api --group crew --version v1 --kind Admiral --controller=true --resource=true --namespaced=false --make=false --pattern=addon
-    unset KUBEBUILDER_ENABLE_PLUGINS
+    $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false
+    $kb create api --group crew --version v1 --kind FirstMate --controller=true --resource=true --make=false
+    $kb create api --group crew --version v1 --kind Admiral --controller=true --resource=true --namespaced=false --make=false
```

Comment:
We have none scenario covering the `create api --plugins=declarative`. Could we add that? Maybe in the project-v3 and project-v2 WDYT

Reply:
Done, lets check if the changes in the corresponding testdata folder are sound.